src/v/config/configuration.cc

```diff
@@ -116,6 +116,24 @@ configuration::configuration()
       "Interval for which all coprocessor offsets are flushed to disk",
       required::no,
       300000ms) // five minutes
+  , enable_v8(
+      *this,
+      "enable_v8",
+      "Enable v8 engine for inline transformation or data-policy",
+      required::no,
+      false)
+  , core_for_executor(
```

Comment:
Why should the core be configurable? It shouldn't make a difference to the user, IMO we should just pick a random core

Reply:
For example, you can set `-c` for redpanda, and choose executor core without seastar. But I do not know, can I get cores with seastar